Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -118,6 +118,21 @@ def audit
       options[:except_cops] = [:FormulaAuditStrict]
     end
 
+    # Run tap audits first
+    if args.tap
+      tap = Tap.fetch(args.tap)
+      ta = TapAuditor.new(tap, strict: args.strict?)
+      ta.audit
+
+      if ta.problems.any?
+        tap_problem_lines = format_problem_lines(ta.problems)
+        tap_problem_count = ta.problems.size
+
+        puts "#{tap.name}:", tap_problem_lines.map { |s| "  #{s}" }
+        odie "#{tap_problem_count} #{"problem".pluralize(tap_problem_count)} in 1 tap detected"
```

Comment:
Why the `odie` here? I think it would be nice to still print all the other problems found if possible.